daemon/api.go

```diff
@@ -296,7 +296,8 @@ func sysInfo(c *Command, r *http.Request, user *auth.UserState) Response {
 			"snap-mount-dir": dirs.SnapMountDir,
 			"snap-bin-dir":   dirs.SnapBinariesDir,
 		},
-		"refresh": refreshInfo,
+		"refresh":             refreshInfo,
+		"classic-unsupported": !dirs.SupportsClassicConfinement(),
```

Comment:
That's a bit too vague for this context. Given that we consider traditional Linux systems to be classic, we have that term reused in a few different places with related but not quite the same meaning.

Perhaps something like this:
```
"confinement-options": ["strict", "classic"]
```
Then, we need to make sure that this is handling core systems correctly, as those will have /snap yet they don't support classic snaps.

Reply:
This is something that can be added once sandbox-features branch lands #5142

Reply:
#5142 has landed, should one of us take over this PR?